core/src/test/scala/c4s/process/ProcessSpec.scala

```diff
@@ -58,6 +58,15 @@ class ProcessSpec extends Specification {
       }.unsafeRunSync()
     }
 
+    "run command reading a stream from another command" >> {
+      withProcess{ implicit shell =>
+        for {
+          result <- Process.run("ls -las")
+          resultStream <- Process.run("wc", result.output)
+        } yield resultStream.exitCode == (ExitCode.Success)
+      }.unsafeRunSync()
+    }
```

Comment:
```suggestion
    "run command reading a stream from another command" >> {
      withProcess { implicit shell =>
        createTmpDirectory[IO].use { path =>
          for {
            _ <- Process.runInPath("touch test-file", path)
            result <- Process.runInPath("ls", path)
            resultStream <- Process.runInPath("wc", result.output, path)
          } yield resultStream.exitCode == (ExitCode.Success)
        }
      }.unsafeRunSync()
    }
```

Even doing that, it would be nice to write something in `test-file` an check `wc` command result so we could see the inputs / outputs are being used properly :thinking: